src/middleware.ts

```diff
@@ -0,0 +1,11 @@
+import { clerkMiddleware } from "@clerk/nextjs/server";
+
+export default clerkMiddleware();
+
+export const config = {
+  matcher: [
+    // Skip Next.js internals and all static files, unless found in search params
+    // "/((?!_next|[^?]*\\.(?:html?|css|js(?!on)|jpe?g|webp|png|gif|svg|ttf|woff2?|ico|csv|docx?|xlsx?|zip|webmanifest)).*)",
```

Comment:
Bypassing the clerk middleware on all routes except the one I actually need it on right now. Running the middleware on all routes was causing the middleware to rewrite the `redirect_url` param to something under the `netlify.app` deploy preview domain. Apparently the AI team faced a similar issue, so i'll check with them when I can to get a proper fix so that the middleware can be run everywhere. For now we can get by without using it.

Working this out cost me so much time today 😭